include/ndarray/pybind11.h

```diff
@@ -55,32 +55,36 @@ template <typename T, int N, int C>
 class type_caster< ndarray::Array<T,N,C> > {
 public:
     bool load(handle src, bool) {
-        ndarray::PyPtr tmp(src.ptr(), true);
+        _src = src; // keep alive for stage 2
```

Comment:
Or maybe it's better to make the `ndarray::PyPtr` the data member, and just construct that here?